programs/stake/src/stake_instruction.rs

```diff
@@ -6458,6 +6459,10 @@ mod tests {
                                 ..Clock::default()
                             }),
                         ),
+                        (
```

Comment:
it's not clear what this addition is for. don't we get stake history from the invoke context / sysvar cache?

Reply:
This to mock StakeHistory account for test. deactivate fn now requires StakeHistory to be available. invoke context/sysvar cache will only have the StakeHistory in the cache when StakeHistory account exists. Therefore, StakeHistory account need to be mocked for the test.

Reply:
Can confirm that this is necessary for our test framework